tests/v2/validation/rbac/rbac_test.go

```diff
@@ -183,6 +187,65 @@ func (rb *RBTestSuite) ValidateRemoveClusterRoles(role string) {
 
 }
 
+func (rb *RBTestSuite) ValidateAddStdUserAsProjectOwner() {
+
+	//Create project as a cluster Owner
+	createProjectAsCO, err := createProject(rb.client, rb.cluster.ID)
+	require.NoError(rb.T(), err)
+	rb.standardUserCOProject = createProjectAsCO
+
+	//Additional testcase1 Validate if cluster owner can add a user as project owner in a project
+	err = users.AddProjectMember(rb.standardUserClient, rb.standardUserCOProject, rb.additionalUser, roleProjectOwner)
+	require.NoError(rb.T(), err)
+	userGetProject, err := projects.GetProjectList(rb.additionalUserClient, rb.cluster.ID)
+	assert.Equal(rb.T(), 1, len(userGetProject.Data))
+	assert.Equal(rb.T(), rb.standardUserCOProject.Name, userGetProject.Data[0].Name)
+
+	err = users.RemoveProjectMember(rb.standardUserClient, rb.additionalUser)
+	require.NoError(rb.T(), err)
+
+}
+
+func (rb *RBTestSuite) ValidateAddMemberAsClusterRoles(role string) {
+
+	//Additional testcase2 Validate if cluster owners should be able to add another standard user as a cluster owner
+	errUserRole := users.AddClusterRoleToUser(rb.standardUserClient, rb.cluster, rb.additionalUser, roleOwner)
+	require.NoError(rb.T(), errUserRole)
+	additionalUserClient, err := rb.additionalUserClient.ReLogin()
+	require.NoError(rb.T(), err)
+	rb.additionalUserClient = additionalUserClient
+
+	clusterList, err := rb.additionalUserClient.Steve.SteveType(clusters.ProvisioningSteveResouceType).ListAll(&types.ListOpts{})
+	require.NoError(rb.T(), err)
+	assert.Equal(rb.T(), 1, len(clusterList.Data))
+
+	err = users.RemoveClusterRoleFromUser(rb.standardUserClient, rb.additionalUser)
+	require.NoError(rb.T(), err)
+
+}
+
+func (rb *RBTestSuite) ValidateAddCMAsProjectOwner(role string) {
+
+	//Additional test3 Cluster owner should be able to add cluster members as a project owner
+
+	errUserRole := users.AddClusterRoleToUser(rb.standardUserClient, rb.cluster, rb.additionalUser, roleMember)
+	require.NoError(rb.T(), errUserRole)
+	additionalUserClient, err := rb.additionalUserClient.ReLogin()
+	require.NoError(rb.T(), err)
+	rb.additionalUserClient = additionalUserClient
+
+	clusterList, err := rb.standardUserClient.Steve.SteveType(clusters.ProvisioningSteveResouceType).ListAll(&types.ListOpts{})
+	require.NoError(rb.T(), err)
+	assert.Equal(rb.T(), 1, len(clusterList.Data))
+
+	//Add the cluster member to the project created by the cluster Owner
+	err = users.AddProjectMember(rb.standardUserClient, rb.standardUserCOProject, rb.additionalUser, roleProjectOwner)
+	require.NoError(rb.T(), err)
+	userGetProject, err := projects.GetProjectList(rb.additionalUserClient, rb.cluster.ID)
+	assert.Equal(rb.T(), rb.standardUserCOProject.Name, userGetProject.Data[0].Name)
+
+}
+
```

Comment:
So Anupama, it seems like the `role` arguments aren't in use, for the new methods. But in the tests, they are already passed. You can either remove hardcoded const roles in the functions by using the `role` arguments or remove those arguments and use roles hardcoded way. @igomez06 what would be the best way for this case?

Reply:
I'm good with either one, if we anticipate the role being dynamic, then let's use the args rather than the consts. But if these test funcs will stay the same, please remove the unused arg. Thank you!